tests/components/modbus/conftest.py

```diff
@@ -62,9 +71,9 @@ async def mock_modbus(hass, caplog, request, do_config):
     config = {
         DOMAIN: [
             {
-                CONF_TYPE: "tcp",
-                CONF_HOST: "modbusTestHost",
-                CONF_PORT: 5501,
+                CONF_TYPE: CONF_TCP,
```

Comment:
Side note: We normally only prefix constants with `CONST_` if they represent a key in the config, ie not if they are values in the config. I'd rename `CONST_TCP` in a follow up.

Reply:
Will do.